package.json

```diff
@@ -11,10 +11,12 @@
     "i18n:report": "vue-cli-service i18n:report --src './src/**/*.?(js|vue)' --locales './src/locales/**/*.json'"
   },
   "dependencies": {
+    "@adapttive/vue-markdown": "^4.0.2",
     "axios": "^1.7.4",
     "bootstrap-vue": "^2.23.1",
     "bowser": "^2.11.0",
     "core-js": "^3.36.1",
+    "dotenv": "^17.2.1",
```

Comment:
Looks like you introduced the `dotenv` dependency. Is that what's necessary in this PR to make the `process.env.VUE_APP_REPROSCHEMA_GITHUB_SRC` work? If not, I would remove it for the sake of a cleaner PR.